browser/resources/settings/brave_sync_page/brave_sync_code_dialog.ts

```diff
@@ -91,7 +91,7 @@ export class SettingsBraveSyncCodeDialogElement extends SettingsBraveSyncCodeDia
     if (!this.syncCode) {
       return 0
     }
-    return this.syncCode.trim().split(' ').length
+    return this.syncCode.trim().split(/\s+/).length
```

Comment:
Having this without any tests is concerning. Can we do the count in core through some common utils header and then export it through `BraveSyncHandler` hence `BraveSyncBrowserProxy` and then Android can share the same util though its UI path?

Reply:
FYI, we used to calculate the password strength in UI per platforms with different implementation then we moved that to brave-core to share the same code.
https://github.com/brave/brave-core/pull/11327

Reply:
>Can we do the count in core through some common utils header and then export it through BraveSyncHandler hence BraveSyncBrowserProxy and then Android can share the same util though its UI path?

thanks, this would be better, as i have similar issue on Android. I am updating the PR.

Reply:
Updated at 94c2685a3c8ffe86537a670a5a56de2470071629